lib/Service/RecordingService.php

```diff
@@ -156,17 +167,23 @@ public function getContentFromFileArray(array $file, Room $room, Participant $pa
 			throw new InvalidArgumentException('invalid_file');
 		}
 
-		$content = file_get_contents($file['tmp_name']);
-		unlink($file['tmp_name']);
```

Comment:
Just to be fully sure, `$file['tmp_name']` is automatically removed by PHP and therefore there is no need to unlink it like it was done before, right?

Reply:
The [official behavior](https://www.php.net/manual/en/features.file-upload.post-method.php) is:

> The file will be deleted from the temporary directory at the end of the request if it has not been moved away or renamed.

In my implementation, neither of those actions (move/rename) is performed on the original temporary file, so it is guaranteed to be removed after the PHP script ends.

An additional `unlink` could be added after [this line](https://github.com/nextcloud/spreed/pull/9253/files#diff-b8455de898ee16a5504ba2177a4674041bf91734999d5405b8ca9cba40cf7eaeR136), but I'm not sure if all the implementations of `file->newFile(...)` guarantee that the file is fully copied after the function call ends.

Reply:
> In my implementation, neither of those actions (move/rename) is performed on the original temporary file, so it is guaranteed to be removed after the PHP script ends.

Sounds good :-)
 
> An additional `unlink` could be added after [this line](https://github.com/nextcloud/spreed/pull/9253/files#diff-b8455de898ee16a5504ba2177a4674041bf91734999d5405b8ca9cba40cf7eaeR136), but I'm not sure if all the implementations of `file->newFile(...)` guarantee that the file is fully copied after the function call ends.

I am not aware of any implementation that does an asynchronous copy. But if there were, I guess it would fail anyway due to the temporal file being deleted at the end of the request. So in practice I think that the implicit unlink is fine (and if it turns out that left over files are eventually found we can always fix it later).

Reply:
Will keep the current implementation then (: